### ISSUE TITLE
[nios_zone] changing a nios_zone does not work

### ISSUE THREAD

**Issue by donestefan** [infobloxopen/infoblox-ansible#60]
I was [asked](https://github.com/ansible-collections/community.general/issues/855#issuecomment-844494744) to move my issue from the ansible-collections repo to the infoblox-ansible repo. So here it is.

##### SUMMARY
changing a parameter of an existing nios_zone always results in an error message:
`"Field is not allowed for update: zone_format", "operation": "update_object"`
This happens even if "zone_format" does not change

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

nios_zone

##### ANSIBLE VERSION

```paste below
ansible 2.9.10
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr  2 2020, 13:16:51) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
```

##### OS / ENVIRONMENT

CentOS Linux release 7.8

##### STEPS TO REPRODUCE

1) apply a playbook like the one below
2) change the ns_group an re-apply

```yaml
- name: "create Forward Zone"
  connection: local
  nios_zone:
    provider: "{{ grid_connector }}"
    restart_if_needed: true
    name: "{{ zone.fqdn }}"
    ns_group: "{{ zone.ns_group}}"
```

##### EXPECTED RESULTS

changes should be applied

##### ACTUAL RESULTS

`"Field is not allowed for update: zone_format", "operation": "update_object"`

**Comment by johansenha** [comment]
Hello, we ran into the same issue when trying to update an existing zone. May this one get a little more priority?

**Comment by anagha-infoblox** [comment]
Hi @donestefan, @johansenha,
This issue has been fixed, and the changes are in the develop branch. It will be merged into the master before the release. 
To install nios_modules from the develop branch: 
ansible-galaxy collection install git+https://github.com/infobloxopen/infoblox-ansible.git#ansible_collections/infoblox/nios_modules,develop

Please do use the same and provide the feedback.

**Comment by anagha-infoblox** [comment]
Hi @donestefan, @johansenha,
This issue has been fixed in the release `v1.1.1`.
